index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Here, this line is obstructing the performance of linters to run an inspection check that helps you to stick to good practices. Kindly remove it. 😉

Reply:
Kudos :clap: on removing this line and manually fixing the linters check :+1: